octopusdeploy/schema_project.go

```diff
@@ -265,14 +342,20 @@ func getProjectSchema() map[string]*schema.Schema {
 			Type:     schema.TypeList,
 		},
 		"release_notes_template": {
+			Computed: true,
 			Optional: true,
 			Type:     schema.TypeString,
 		},
 		"slug": {
 			Computed: true,
 			Type:     schema.TypeString,
 		},
-		"space_id": getSpaceIDSchema(),
+		"space_id": {
+			Description:      "The space ID associated with this project.",
+			Required:         true,
```

Comment:
I'm still wondering whether forcing this to be explicit/required is the right path.